
<TranslatedText
	xmlns:cws="http://schemas.cordys.com/cws/1.0"
	type="com.cordys.translation.TranslatedText"
	RuntimeDocumentID="B4B676CD-53D8-A1E9-B1D8-937B249888BD">
	<DocumentID>B4B676CD-53D8-A1E9-B1D8-937B249848BD</DocumentID>
	<Name>B4B676CD-53D8-A1E9-B1D8-937B249848BD</Name>
	<Description>Catch Exception</Description>
	<CreatedBy>sysadmin</CreatedBy>
	<CreationDate>2019-08-25T18:03:22.991</CreationDate>
	<Notes/>
	<Text>Catch Exception</Text>
	<Language>
		<uri
			id="B4B676CD-53D8-A1E9-B0F5-CC8A88E208BD"
		/>
	</Language>
	<TextIdentifier>
		<uri
			id="B4B676CD-53D8-A1E9-B1D8-937B2497C8BD"
		/>
	</TextIdentifier>
</TranslatedText>
